cmd/bucket-handlers_test.go

```diff
@@ -369,7 +369,7 @@ func testListMultipartUploadsHandler(obj ObjectLayer, instanceType, bucketName s
 			maxUploads:         "0",
 			accessKey:          credentials.AccessKey,
 			secretKey:          credentials.SecretKey,
-			expectedRespStatus: http.StatusNotFound,
```

Comment:
why is this changed? a non-existent bucket must return 404

Reply:
Test were failing as minio returning 200 OK in this case. I feel it's because we return from cache now.